### PR TITLE
[SC-373] Add a cost center tag option

### PR DESCRIPTION
**Pull request by zaro0508**
Add a service catalog CostCenter tag.  It is a required tag so users
will need to pick one cost center from the list when provisioning SC
products.  The source of the cost center tags are in the
aws-infra repo[1].

[1] https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/tags/SageFinancialProgramCodes.json

### CODE REVIEW

File: sceptre/scipool/config/bmgfki/sc-tag-options.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ dependencies:
 sceptre_user_data:
   Departments: !file sc-tag-options/internal/Departments.json
   Projects: !file sc-tag-options/internal/Projects.json
+  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
```
Comment (brucehoff) [inline sceptre/scipool/config/bmgfki/sc-tag-options.yaml:16]: BMGF-KI (as well as STRIDES) should not ask for a Sage cost center.

File: sceptre/scipool/config/strides/sc-tag-options.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ dependencies:
 sceptre_user_data:
   Departments: !file sc-tag-options/internal/Departments.json
   Projects: !file sc-tag-options/internal/Projects.json
+  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
```
Comment (brucehoff) [inline sceptre/scipool/config/strides/sc-tag-options.yaml:16]: STRIDES products (like BMGF-Ki products) should not ask for a Sage Cost Center.

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: sceptre/scipool/config/bmgfki/sc-tag-options.yaml (hunk 1 above)
Comment (zaro0508) [reply sceptre/scipool/config/bmgfki/sc-tag-options.yaml]: fixed

File: sceptre/scipool/config/strides/sc-tag-options.yaml (hunk 2 above)
Comment (zaro0508) [reply sceptre/scipool/config/strides/sc-tag-options.yaml]: fixed

**Review by brucehoff — APPROVED** [review]

**Pull request merged**